src/preset_cli/api/clients/preset.py

```diff
@@ -242,3 +242,17 @@ def get_base_url(self, version: Optional[str] = "v1") -> URL:
         Return the base URL for API calls.
         """
         return self.baseurl / version
+
+    def get_group_membership(
+        self,
+        team_name: str,
+        page: int,
+    ) -> json:
```

Comment:
ohh I didn't know that - thanks!

Reply:
In this case, I believe I need to add `Dict` here?
```
from typing import Any, Iterator, List, Optional, Union
```

Going to add on my commit just in case.